arkouda/strings.py

```diff
@@ -1043,3 +1045,332 @@ def unregister_strings_by_name(user_defined_name : str) -> None:
         """
         unregister_pdarray_by_name(f"{user_defined_name}.bytes")
         unregister_pdarray_by_name(f"{user_defined_name}.offsets")
+
+class SArrays:
+    """
+    Represents an array of (suffix) arrays whose data resides on the arkouda server.
```

Comment:
the `SArrays` class currently resides in the `strings.py` file. In my opinion it should be separated into it's own file

Reply:
Yes, I agree!